### DISCUSSION TITLE
[Support] Multiwan Wireguard

### DISCUSSION THREAD

**Discussion by olafhuerta97** [pivpn/pivpn#1453] [Support]
## In raising this issue, I confirm the following:

`{please fill the checkboxes, e.g: [X]}`

- [x] I have read the [documentation](https://docs.pivpn.io)
- [x] I have read and understood the [contributors guide](https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting can be *is* directly related to the pivpn installer script. (Yeah, this is not but I do not know where else to ask, browsing on the web I did not find really any useful information, or I did not undestand it haha 😪)

### Have you searched for similar issues and solutions?
    (yes/no / which issues?)
yes, I did not find any. I think I saw one with same issue but on OpenVPN and now I cannot find it 

### Has your install failed? 
```
no
```

#### What type of hardware are you running PiVPN at?

```
 Raspberry Pi 4 Model B Rev 1.4 
```
#### Output of `uname -a`
```
Linux raspberrypi 5.10.63-v7l+ #1496 SMP Wed Dec 1 15:58:56 GMT 2021 armv7l GNU/Linux 
```

#### Output of `cat /etc/os-release`
```
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"                                                                            NAME="Raspbian GNU/Linux"                                                                                               VERSION_ID="10"                                                                                                         VERSION="10 (buster)"                                                                                                   VERSION_CODENAME=buster                                                                                                 ID=raspbian                                                                                                             ID_LIKE=debian                                                                                                          HOME_URL="http://www.raspbian.org/"                                                                                     SUPPORT_URL="http://www.raspbian.org/RaspbianForums"                                                                    BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"  
```

### If install failed Please provide the console output of  `curl -L https://install.pivpn.io | bash`
```
did not failed
```

### Issue

Hi, I know this is more a request for help of configuration of wireguard or so, but is it possible to manage 2 ddns with the same instance of wireguard. 
I have to ISP and therefor 2 wan, I have 2 dynamic dns and I want wireguard to listen on both, I dont care if they are in different virtual lan but I really would like some backup or something.
Also I can open the ports to Pi, but I really did not find where to start, has somebody made somithing like this? 
Is there a way that my same client works on either wan-vpn?
Thank you!

**Comment by coolapso** [comment]
I am not sure how you can get that done. 

Moving this to discussions, hang on thigh maybe someone might have an idea on how to do it.
